src/devtools/client/debugger/src/components/Editor/LineHitCounts.module.css

```diff
@@ -3,19 +3,26 @@
   align-items: center;
   justify-content: center;
 
-  width: 26px;
-  height: var(--theme-code-font-size);
-  margin: 2px;
-  padding-top: 1px;
+  width: 100%;
+  height: calc(var(--theme-code-font-size) * 1.4);
+  position: relative;
+  top: -0.25px;
```

Comment:
What's this? 😅

Reply:
Same as above, it's making sure the background covers a larger height so there isn't any space between the markers. We then need to pull things back up a little to accommodate for the larger height 😅